### PR TITLE
feat(qwik-pod): Fix file viewer not being populated

### PR DESCRIPTION
**Pull request by SDaian**
## Description

- [x] FileViewer is not being populated with the content from the file

Closes: #1072 

### Screenshots:

image

### CODE REVIEW

File: qwik-graphql-tailwind/src/components/file-viewer/index.tsx [hunk 1]
```diff
@@ -47,7 +46,7 @@ export const FileViewer = component$(() => {
     return <div>Loading...</div>;
   }
 
-  const extension = path?.split('.').pop();
+  const extension = path?.split('.').pop()?.split('/').shift();
```
Comment (vyktoremario) [inline qwik-graphql-tailwind/src/components/file-viewer/index.tsx:49]: I am trying to understand why an additional `split()` and `shif()` was added.
Is there a scenario where `.split('.').pop()` doesn't work?

**Review by vyktoremario — COMMENTED** [review]
Left a question

File: qwik-graphql-tailwind/src/components/file-viewer/index.tsx (hunk 1 above)
Comment (SDaian) [reply qwik-graphql-tailwind/src/components/file-viewer/index.tsx:49]: is kind complicated but if the url has `.yml/` need only the `.yml` not the `/`, that's why.

Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/file-viewer/index.tsx:49]: this ticket also handle this issues https://github.com/thisdot/starter.dev-github-showcases/issues/1085

**Review by vyktoremario — APPROVED** [review]

**Comment by SDaian** [comment]
This issue was fixed on [#1085]( https://github.com/thisdot/starter.dev-github-showcases/issues/1085)

**Pull request closed without merging**